prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -19,10 +19,29 @@ postsubmits:
           secretName: compute-image-tools-test-service-account
     max_concurrency: 10
     branches:
-      - ^master$
-
+    - ^master$
 presubmits:
   GoogleCloudPlatform/osconfig:
+  - name: osconfig-presubmit-build
+    cluster: gcp-guest
+    trigger: "(?m)^/osconfigbuild$"
+    rerun_command: "/osconfigbuild"
+    context: prow/presubmit/gotest
```

Comment:
this is the name that will appear in github status update. suggest prow/presubmit/daisy or prow/presubmit/packagebuild

Reply:
does it make sense to make specifiic to osconfig-agent-build?

prow/presubmit/osconfig-agent-build

Reply:
no, i don't think it should, the logic isn't specific to osconfig. naming it so it makes sense what it's trying to do is clearest imo. `packagebuild` is probably best

Reply:
as discussed offline, changing the name